package.json

```diff
@@ -13,7 +13,8 @@
     "precommit": "yarn check && yarn build && lint-staged",
     "prepublish": "yarn build",
     "release": "np",
-    "test": "jest"
+    "test": "jest ./tests",
+    "e2e": "jest ./e2e"
```

Comment:
I am *so* not sure about this.

Reply:
Why?

The only thing I'd change is moving the regular tests to `<file>.spec.js` and have it besides the file it tests.

In that cased we'd need to change the Jest config and have two different ones for e2e and unit tests.

Reply:
I'm not sure I understand. You mean having something like:

```
lib/some-module.js
lib/some-module.spec.js
helpers/some-helper.js
helpers/some-helper.spec.js
```

I'm ok with that, except that it would create a lot of `__snapthots__` directories, wouldn't it?

Reply:
Yeah, that's right. Anyways, not related to this PR.